app/models/comment.rb

```diff
@@ -1,4 +1,15 @@
 class Comment < ApplicationRecord
-  belongs_to :author
-  belongs_to :post
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post, foreign_key: 'post_id'
```

Comment:
[REQUIRED] Kindly remove the foreign_key in the post as it is not needed. And for `belongs_to :user` it should look like this: `belongs_to :author, class_name: 'User'`.

Reply:
Done! thanks!